hugegraph-core/src/main/java/com/baidu/hugegraph/util/collection/Int2IntsMap.java

```diff
@@ -0,0 +1,177 @@
+
+
+    /*
+     *  chunkMap         chunkTable
+     *
+     *   --------              ---------------
+     *  | 1 | 0  |--------->0 |      33       | (nextPosition)
+     *  | 2 | 10 |-----+    1 |      19       | (size)
+     *  | 3 | 40 |---+ |    2 |   int data    |
+     *  | . | .  |   | |    3 |   int data    |
+     *  | x | y  |   | |    . |     ...       |
+     *   --------    | |    9 |      20       |-----------------+
+     *               | |       ---------------                  |
+     *               | +-->10 |     13        | (nextPosition)  |
+     *               |     11 |      1        | (size)          |
+     *               |     12 |  int data     |                 | nextChunk
+     *               |     13 |      0        |                 |
+     *               |      . |     ...       |                 |
+     *               |     19 |      0        |                 |
+     *               |         ---------------                  |
+     *               |     20 |   int data    |<----------------+
+     *               |     21 |   int data    |
+     *               |     22 |   int data    |
+     *               |     23 |   int data    |
+     *               |      . |     ...       |
+     *               |     29 |      30       |-----------------+
+     *               |         ---------------                  | nextChunk
+     *               |     30 |   int data    |<----------------+
+     *               |     31 |   int data    |
+     *               |     32 |   int data    |
+     *               |     33 |      0        |
+     *               |      . |     ...       |
+     *               |     39 |      0        |
+     *               |         ---------------
+     *               +---->40 |      48       | (nextPosition)
+     *                     41 |       6       | (size)
+     *                     42 |   int data    |
+     *                     43 |   int data    |
+     *                      . |     ...       |
+     *                     47 |   int data    |
+     *                     48 |      0        |
+     *                     49 |      0        |
+     *                         ---------------
+     *                     50 |     ...       |
+     *                        |     ...       |
+     *                        |     ...       |
+     *                        |     ...       |
+     *
+     *
+     */
+
+    private IntIntHashMap chunkMap;
+    private int[] chunkTable;
+
+    private int nextChunk;
+
+    public Int2IntsMap() {
+        this.chunkMap = new IntIntHashMap(INIT_KEY_CAPACITY);
+        this.chunkTable = new int[INIT_KEY_CAPACITY * CHUNK_SIZE];
+        this.nextChunk = 0;
+    }
+
+    public void add(int key, int value) {
+        if (this.chunkMap.containsKey(key)) {
+            int firstChunk = this.chunkMap.get(key);
+            int nextPosition = this.chunkTable[firstChunk + OFFSET_POSITION];
```

Comment:
the meaning of position is generic, improve the variable naming to nextXxPointer.
also address other comments:
https://github.com/hugegraph/hugegraph/pull/1409/files/75b2ef6c4de64306b75d7b91d1672c32e057360b..7248e77a062d8e375f2ef85e06ec16d5b5373337#diff-c6fe21c746e89abdbac5015ba5d5b3ec346aef2dd756922bac904958cd2aabea